source/documentation/troubleshooting/password.md

```diff
@@ -0,0 +1,5 @@
+## Forgetten passwords
```

Comment:
`s/Forgetten/Forgotten/`

Reply:
Fixed. The link was also wrong. It now points to `/forgot_password`, not `/password_reset`.